--- crds.go.original	2025-11-24 00:31:48.100593919 +0800
+++ crds.go.modified	2025-11-24 00:39:41.207252473 +0800
@@ -3,9 +3,9 @@
 //
 //  SPDX-License-Identifier: MIT
 
-// Package v1alpha1 contains API Schema definitions for the trusted-execution-clusters v1alpha1 API group.
+// Package v1alpha1 contains API Schema definitions for the confidential-clusters v1alpha1 API group.
 // +kubebuilder:object:generate=true
-// +groupName=trusted-execution-clusters.io
+// +groupName=confidential-clusters.io
 package v1alpha1
 
 import (
@@ -16,7 +16,7 @@
 
 var (
 	// GroupVersion is group version used to register these objects.
-	GroupVersion = schema.GroupVersion{Group: "trusted-execution-clusters.io", Version: "v1alpha1"}
+	GroupVersion = schema.GroupVersion{Group: "confidential-clusters.io", Version: "v1alpha1"}
 
 	// SchemeBuilder is used to add go types to the GroupVersionKind scheme.
 	SchemeBuilder = &scheme.Builder{GroupVersion: GroupVersion}
@@ -30,13 +30,13 @@
 // +kubebuilder:rbac:groups="",resources=secrets,verbs=create
 // +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;create;update
 // +kubebuilder:rbac:groups=batch,resources=jobs,verbs=create;delete;list;watch
-// +kubebuilder:rbac:groups=trusted-execution-clusters.io,resources=trustedexecutionclusters,verbs=list;watch
-// +kubebuilder:rbac:groups=trusted-execution-clusters.io,resources=trustedexecutionclusters/status,verbs=patch
-// +kubebuilder:rbac:groups=trusted-execution-clusters.io,resources=machines,verbs=create;list;delete;watch
+// +kubebuilder:rbac:groups=confidential-clusters.io,resources=confidentialclusters,verbs=list;watch
+// +kubebuilder:rbac:groups=confidential-clusters.io,resources=confidentialclusters/status,verbs=patch
+// +kubebuilder:rbac:groups=confidential-clusters.io,resources=machines,verbs=create;list;delete;watch
 
-// TrustedExecutionClusterSpec defines the desired state of TrustedExecutionCluster
+// ConfidentialClusterSpec defines the desired state of ConfidentialCluster
 // +kubebuilder:validation:XValidation:rule="!has(oldSelf.publicTrusteeAddr) || has(self.publicTrusteeAddr)", message="Value is required once set"
-type TrustedExecutionClusterSpec struct {
+type ConfidentialClusterSpec struct {
 	// Image reference to Trustee all-in-one image
   // +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
 	TrusteeImage *string `json:"trusteeImage"`
@@ -65,8 +65,8 @@
 	RegisterServerPort int32 `json:"registerServerPort,omitempty"`
 }
 
-// TrustedExecutionClusterStatus defines the observed state of TrustedExecutionCluster.
-type TrustedExecutionClusterStatus struct {
+// ConfidentialClusterStatus defines the observed state of ConfidentialCluster.
+type ConfidentialClusterStatus struct {
 	// +listType=map
 	// +listMapKey=type
 	// +optional
@@ -76,30 +76,30 @@
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 
-// TrustedExecutionCluster is the Schema for the trustedexecutionclusters API
-type TrustedExecutionCluster struct {
+// ConfidentialCluster is the Schema for the confidentialclusters API
+type ConfidentialCluster struct {
 	metav1.TypeMeta `json:",inline"`
 
 	// metadata is a standard object metadata
 	// +optional
 	metav1.ObjectMeta `json:"metadata,omitzero"`
 
-	// spec defines the desired state of TrustedExecutionCluster
+	// spec defines the desired state of ConfidentialCluster
 	// +required
-	Spec TrustedExecutionClusterSpec `json:"spec"`
+	Spec ConfidentialClusterSpec `json:"spec"`
 
-	// status defines the observed state of TrustedExecutionCluster
+	// status defines the observed state of ConfidentialCluster
 	// +optional
-	Status TrustedExecutionClusterStatus `json:"status,omitzero"`
+	Status ConfidentialClusterStatus `json:"status,omitzero"`
 }
 
 // +kubebuilder:object:root=true
 
-// TrustedExecutionClusterList contains a list of TrustedExecutionCluster
-type TrustedExecutionClusterList struct {
+// ConfidentialClusterList contains a list of ConfidentialCluster
+type ConfidentialClusterList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata"`
-	Items           []TrustedExecutionCluster `json:"items"`
+	Items           []ConfidentialCluster `json:"items"`
 }
 
 // MachineSpec defines the desired state of Machine
