--- confidentialcluster_viewer_role.yaml.original	2025-11-24 00:49:09.811679090 +0800
+++ confidentialcluster_viewer_role.yaml.modified	2025-11-24 00:50:40.061800416 +0800
@@ -5,7 +5,7 @@
 # This rule is not used by the project trusted-cluster-operator itself.
 # It is provided to allow the cluster admin to help manage permissions for users.
 #
-# Grants read-only access to trusted-execution-clusters.io resources.
+# Grants read-only access to confidential-clusters.io resources.
 # This role is intended for users who need visibility into these resources
 # without permissions to modify them. It is ideal for monitoring purposes and limited-access viewing.
 
@@ -18,7 +18,7 @@
   name: trustedexecutioncluster-viewer-role
 rules:
 - apiGroups:
-  - trusted-execution-clusters.io
+  - confidential-clusters.io
   resources:
   - trustedexecutionclusters
   verbs:
@@ -26,7 +26,7 @@
   - list
   - watch
 - apiGroups:
-  - trusted-execution-clusters.io
+  - confidential-clusters.io
   resources:
   - trustedexecutionclusters/status
   verbs:
