--- main.rs.after	2025-11-24 00:58:09.669406409 +0800
+++ main.rs.before	2025-11-24 00:56:50.235299036 +0800
@@ -201,7 +201,7 @@
                 ..Default::default()
             },
             PolicyRule {
-                api_groups: Some(vec![Machine::group(&()).to_string()]),
+                api_groups: Some(vec!["confidential-containers.io".to_string()]),
                 resources: Some(vec!["machines".to_string()]),
                 verbs: vec![
                     "create".to_string(),
